data/brands/shop/department_store.json

```
@@ -1386,6 +1436,17 @@
         "shop": "department_store"
       }
     },
+    {
+      "displayName": "World Duty Free",
+      "id": "worlddutyfree-b0f55b",
+      "locationSet": {"include": ["it"]},
```

```suggestion
      "locationSet": {"include": ["001"]},
```
According to their [Wikipedia article](https://en.wikipedia.org/wiki/World_Duty_Free#World_Duty_Free_Group), they have locations in 20 countries